core/src/main/scala/io/qbeast/core/transform/Transformation.scala

```diff
@@ -53,6 +54,9 @@ case class IdentityTransformation(newVal: Any) extends Transformation {
   override def transform(value: Any): Double = value match {
     case v: Number =>
       v.byteValue()
+    // Not sure that we need this
+    case v: Timestamp =>
+      v.getTime().byteValue()
```

Comment:
Try to test a case with a single timestamp. It will fail now, but the resolution of #128 could give us a hint on this.

Reply:
I get the same error as in Issue [#128](https://github.com/Qbeast-io/qbeast-spark/issues/128). Should we create a new PR to solve it or can we try to solve it in this one?

Reply:
Yes, I will create a new PR. Hope we can merge it today and you can synch quickly.